charts/incubator/minio/Chart.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: v2
+appVersion: "0.12.2"
+dependencies:
+- name: common
+  repository: https://truecharts.org
+  version: 8.5.4
+description: Minio server
```

Give an actual description of the App:

```suggestion
description: Minio is a self-hosted S3 storage server
```